[2024-04-14T08:45:03.200+0000] {processor.py:154} INFO - Started process (PID=14354) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:03.208+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:45:03.210+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:45:03.210+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:03.240+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:45:03.237+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:45:03.242+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:03.282+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.091 seconds
[2024-04-14T08:45:33.913+0000] {processor.py:154} INFO - Started process (PID=14422) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:33.918+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:45:33.920+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:45:33.920+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:33.947+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:45:33.943+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:45:33.950+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:45:33.985+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.083 seconds
[2024-04-14T08:46:04.646+0000] {processor.py:154} INFO - Started process (PID=14499) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:04.655+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:46:04.658+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:46:04.658+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:04.683+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:46:04.679+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:46:04.684+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:04.718+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-14T08:46:34.883+0000] {processor.py:154} INFO - Started process (PID=14567) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:34.892+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:46:34.894+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:46:34.894+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:34.913+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:46:34.910+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:46:34.915+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:46:34.956+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.080 seconds
[2024-04-14T08:47:05.867+0000] {processor.py:154} INFO - Started process (PID=14635) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:05.869+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:47:05.873+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:47:05.872+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:05.960+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:47:05.894+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:47:05.965+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:06.029+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.173 seconds
[2024-04-14T08:47:36.259+0000] {processor.py:154} INFO - Started process (PID=14712) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:36.272+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:47:36.274+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:47:36.273+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:36.301+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:47:36.298+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:47:36.303+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:47:36.340+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-14T08:48:07.459+0000] {processor.py:154} INFO - Started process (PID=14775) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:07.468+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:48:07.477+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:48:07.476+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:07.501+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:48:07.498+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:48:07.502+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:07.541+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.118 seconds
[2024-04-14T08:48:38.743+0000] {processor.py:154} INFO - Started process (PID=14843) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:38.749+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:48:38.751+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:48:38.751+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:38.782+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:48:38.776+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:48:38.785+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:48:38.830+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.095 seconds
[2024-04-14T08:49:08.908+0000] {processor.py:154} INFO - Started process (PID=14908) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:08.910+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:49:08.913+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:49:08.912+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:08.942+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:49:08.936+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:49:08.944+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:08.981+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.082 seconds
[2024-04-14T08:49:39.210+0000] {processor.py:154} INFO - Started process (PID=14984) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:39.212+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:49:39.214+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:49:39.214+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:39.236+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:49:39.233+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:49:39.239+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:49:39.278+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.077 seconds
[2024-04-14T08:50:11.470+0000] {processor.py:154} INFO - Started process (PID=15052) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:11.485+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:50:11.487+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:50:11.486+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:11.537+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:50:11.527+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:50:11.539+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:11.623+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.166 seconds
[2024-04-14T08:50:42.153+0000] {processor.py:154} INFO - Started process (PID=15119) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:42.167+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:50:42.175+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:50:42.174+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:42.197+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:50:42.193+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:50:42.198+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:50:42.252+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.110 seconds
[2024-04-14T08:51:12.375+0000] {processor.py:154} INFO - Started process (PID=15185) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:12.485+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:51:12.492+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:51:12.489+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:12.520+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:51:12.517+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:51:12.521+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:12.578+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.213 seconds
[2024-04-14T08:51:42.890+0000] {processor.py:154} INFO - Started process (PID=15253) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:42.892+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:51:42.895+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:51:42.894+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:42.911+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:51:42.908+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:51:42.913+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:51:42.948+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.064 seconds
[2024-04-14T08:52:13.237+0000] {processor.py:154} INFO - Started process (PID=15330) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:13.239+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:52:13.241+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:52:13.241+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:13.257+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:52:13.255+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:52:13.258+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:13.293+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-14T08:52:43.524+0000] {processor.py:154} INFO - Started process (PID=15396) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:43.527+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:52:43.529+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:52:43.529+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:43.553+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:52:43.549+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:52:43.557+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:52:43.606+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.092 seconds
[2024-04-14T08:53:14.256+0000] {processor.py:154} INFO - Started process (PID=15463) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:14.262+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:53:14.267+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:53:14.267+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:14.291+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:53:14.287+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:53:14.294+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:14.379+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.135 seconds
[2024-04-14T08:53:44.610+0000] {processor.py:154} INFO - Started process (PID=15535) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:44.614+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:53:44.621+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:53:44.620+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:44.674+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:53:44.662+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:53:44.677+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:53:44.748+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.155 seconds
[2024-04-14T08:54:14.952+0000] {processor.py:154} INFO - Started process (PID=15604) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:14.962+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:54:14.968+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:54:14.967+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:15.026+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:54:15.014+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:54:15.030+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:15.143+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.207 seconds
[2024-04-14T08:54:45.787+0000] {processor.py:154} INFO - Started process (PID=15672) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:45.796+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:54:45.798+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:54:45.798+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:45.826+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:54:45.821+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:54:45.829+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:54:45.885+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.112 seconds
[2024-04-14T08:55:16.074+0000] {processor.py:154} INFO - Started process (PID=15739) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:16.079+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:55:16.087+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:55:16.087+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:16.145+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:55:16.139+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:55:16.153+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:16.240+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.187 seconds
[2024-04-14T08:55:46.425+0000] {processor.py:154} INFO - Started process (PID=15807) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:46.430+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:55:46.436+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:55:46.436+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:46.464+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:55:46.461+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:55:46.467+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:55:46.526+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.109 seconds
[2024-04-14T08:56:17.061+0000] {processor.py:154} INFO - Started process (PID=15882) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:17.064+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:56:17.066+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:56:17.066+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:17.090+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:56:17.086+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:56:17.091+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:17.134+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.089 seconds
[2024-04-14T08:56:47.869+0000] {processor.py:154} INFO - Started process (PID=15950) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:47.872+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:56:47.874+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:56:47.874+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:47.901+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:56:47.897+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:56:47.903+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:56:47.944+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.087 seconds
[2024-04-14T08:57:19.176+0000] {processor.py:154} INFO - Started process (PID=16026) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:19.180+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:57:19.188+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:57:19.185+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:19.220+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:57:19.214+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:57:19.224+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:19.271+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.112 seconds
[2024-04-14T08:57:49.409+0000] {processor.py:154} INFO - Started process (PID=16092) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:49.414+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:57:49.416+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:57:49.416+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:49.447+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:57:49.444+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:57:49.449+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:57:49.490+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.091 seconds
[2024-04-14T08:58:20.282+0000] {processor.py:154} INFO - Started process (PID=16167) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:20.287+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:58:20.290+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:58:20.290+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:20.318+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:58:20.315+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:58:20.320+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:20.370+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.102 seconds
[2024-04-14T08:58:50.569+0000] {processor.py:154} INFO - Started process (PID=16234) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:50.572+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:58:50.574+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:58:50.574+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:50.595+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:58:50.592+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:58:50.597+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:58:50.633+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.073 seconds
[2024-04-14T08:59:21.136+0000] {processor.py:154} INFO - Started process (PID=16313) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:21.141+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:59:21.142+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:59:21.142+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:21.163+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:59:21.160+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:59:21.164+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:21.208+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-14T08:59:51.564+0000] {processor.py:154} INFO - Started process (PID=16382) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:51.571+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T08:59:51.578+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:59:51.578+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:51.614+0000] {logging_mixin.py:120} INFO - [2024-04-14T08:59:51.604+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T08:59:51.616+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T08:59:51.671+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.123 seconds
[2024-04-14T09:00:22.196+0000] {processor.py:154} INFO - Started process (PID=16447) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:22.200+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:00:22.203+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:00:22.202+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:22.237+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:00:22.234+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:00:22.239+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:22.276+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.092 seconds
[2024-04-14T09:00:52.857+0000] {processor.py:154} INFO - Started process (PID=16524) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:52.862+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:00:52.866+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:00:52.865+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:52.903+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:00:52.899+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:00:52.905+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:00:52.963+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.117 seconds
[2024-04-14T09:01:23.317+0000] {processor.py:154} INFO - Started process (PID=16592) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:23.325+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:01:23.328+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:01:23.327+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:23.352+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:01:23.348+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:01:23.353+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:23.393+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-14T09:01:53.644+0000] {processor.py:154} INFO - Started process (PID=16658) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:53.649+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:01:53.653+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:01:53.652+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:53.713+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:01:53.708+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:01:53.718+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:01:53.852+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.228 seconds
[2024-04-14T09:02:23.980+0000] {processor.py:154} INFO - Started process (PID=16731) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:23.984+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:02:23.989+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:02:23.988+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:24.036+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:02:24.026+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:02:24.038+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:24.087+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.113 seconds
[2024-04-14T09:02:54.608+0000] {processor.py:154} INFO - Started process (PID=16798) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:54.610+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:02:54.613+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:02:54.612+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:54.634+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:02:54.631+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:02:54.635+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:02:54.667+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.066 seconds
[2024-04-14T09:03:25.006+0000] {processor.py:154} INFO - Started process (PID=16865) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:25.013+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:03:25.016+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:03:25.016+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:25.042+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:03:25.039+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:03:25.048+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:25.093+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.102 seconds
[2024-04-14T09:03:55.321+0000] {processor.py:154} INFO - Started process (PID=16943) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:55.323+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:03:55.325+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:03:55.325+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:55.343+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:03:55.340+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:03:55.346+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:03:55.382+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.071 seconds
[2024-04-14T09:04:25.539+0000] {processor.py:154} INFO - Started process (PID=17011) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:25.541+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:04:25.542+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:04:25.542+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:25.573+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:04:25.570+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:04:25.575+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:25.619+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.089 seconds
[2024-04-14T09:04:55.948+0000] {processor.py:154} INFO - Started process (PID=17089) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:55.951+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:04:55.953+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:04:55.953+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:55.975+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:04:55.972+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:04:55.978+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:04:56.013+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.070 seconds
[2024-04-14T09:05:26.713+0000] {processor.py:154} INFO - Started process (PID=17156) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:26.717+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:05:26.719+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:05:26.719+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:26.741+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:05:26.738+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:05:26.743+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:26.788+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.088 seconds
[2024-04-14T09:05:57.513+0000] {processor.py:154} INFO - Started process (PID=17235) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:57.516+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:05:57.519+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:05:57.519+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:57.544+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:05:57.538+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:05:57.546+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:05:57.594+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.087 seconds
[2024-04-14T09:06:28.988+0000] {processor.py:154} INFO - Started process (PID=17304) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:28.992+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:06:29.001+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:06:28.999+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:29.063+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:06:29.058+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:06:29.070+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:29.348+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.388 seconds
[2024-04-14T09:06:59.827+0000] {processor.py:154} INFO - Started process (PID=17371) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:59.829+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:06:59.833+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:06:59.833+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:59.864+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:06:59.858+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:06:59.865+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:06:59.936+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.121 seconds
[2024-04-14T09:07:30.005+0000] {processor.py:154} INFO - Started process (PID=17439) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:07:30.007+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:07:30.012+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:07:30.010+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:07:30.039+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:07:30.035+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:07:30.043+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:07:30.123+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.127 seconds
[2024-04-14T09:08:01.776+0000] {processor.py:154} INFO - Started process (PID=17506) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:01.781+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:08:01.787+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:08:01.784+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:01.832+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:08:01.828+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:08:01.837+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:01.937+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.176 seconds
[2024-04-14T09:08:32.266+0000] {processor.py:154} INFO - Started process (PID=17575) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:32.272+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:08:32.274+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:08:32.274+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:32.308+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:08:32.301+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:08:32.311+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:08:32.378+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.124 seconds
[2024-04-14T09:09:04.336+0000] {processor.py:154} INFO - Started process (PID=17650) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:04.340+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:09:04.343+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:09:04.342+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:04.368+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:09:04.360+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:09:04.370+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:04.418+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.090 seconds
[2024-04-14T09:09:35.980+0000] {processor.py:154} INFO - Started process (PID=17714) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:35.984+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:09:35.992+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:09:35.991+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:36.025+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:09:36.022+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:09:36.027+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:09:36.128+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.162 seconds
[2024-04-14T09:10:06.523+0000] {processor.py:154} INFO - Started process (PID=17780) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:06.526+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:10:06.528+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:10:06.528+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:06.563+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:10:06.554+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:10:06.565+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:06.609+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.112 seconds
[2024-04-14T09:10:37.118+0000] {processor.py:154} INFO - Started process (PID=17857) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:37.121+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:10:37.124+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:10:37.124+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:37.148+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:10:37.145+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:10:37.150+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:10:37.198+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.089 seconds
[2024-04-14T09:11:07.736+0000] {processor.py:154} INFO - Started process (PID=17926) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:07.737+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:11:07.739+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:11:07.739+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:07.767+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:11:07.762+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:11:07.769+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:07.811+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.085 seconds
[2024-04-14T09:11:38.115+0000] {processor.py:154} INFO - Started process (PID=18004) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:38.119+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:11:38.124+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:11:38.123+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:38.149+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:11:38.145+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:11:38.150+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:11:38.200+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.098 seconds
[2024-04-14T09:12:08.358+0000] {processor.py:154} INFO - Started process (PID=18069) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:08.366+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:12:08.368+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:12:08.367+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:08.402+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:12:08.398+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:12:08.404+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:08.448+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.099 seconds
[2024-04-14T09:12:38.936+0000] {processor.py:154} INFO - Started process (PID=18147) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:38.939+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:12:38.941+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:12:38.941+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:38.978+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:12:38.974+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:12:38.982+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:12:39.043+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.118 seconds
[2024-04-14T09:13:09.659+0000] {processor.py:154} INFO - Started process (PID=18217) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:09.665+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:13:09.675+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:13:09.675+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:09.762+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:13:09.755+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:13:09.765+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:09.880+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.240 seconds
[2024-04-14T09:13:40.020+0000] {processor.py:154} INFO - Started process (PID=18287) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:40.027+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:13:40.031+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:13:40.030+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:40.059+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:13:40.053+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:13:40.061+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:13:40.116+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.105 seconds
[2024-04-14T09:14:10.619+0000] {processor.py:154} INFO - Started process (PID=18364) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:10.622+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:14:10.624+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:14:10.624+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:10.654+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:14:10.651+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:14:10.655+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:10.710+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.098 seconds
[2024-04-14T09:14:40.856+0000] {processor.py:154} INFO - Started process (PID=18431) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:40.871+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:14:40.880+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:14:40.879+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:40.955+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:14:40.937+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:14:40.957+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:14:41.063+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.230 seconds
[2024-04-14T09:15:11.420+0000] {processor.py:154} INFO - Started process (PID=18509) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:11.429+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:15:11.433+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:15:11.433+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:11.474+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:15:11.465+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:15:11.479+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:11.565+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.167 seconds
[2024-04-14T09:15:41.914+0000] {processor.py:154} INFO - Started process (PID=18578) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:41.919+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:15:41.922+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:15:41.922+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:41.948+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:15:41.943+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:15:41.950+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:15:41.982+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.079 seconds
[2024-04-14T09:16:12.207+0000] {processor.py:154} INFO - Started process (PID=18647) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:16:12.209+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:16:12.211+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:16:12.211+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:16:12.228+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:16:12.225+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:16:12.230+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:16:12.273+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.072 seconds
[2024-04-14T09:28:03.271+0000] {processor.py:154} INFO - Started process (PID=18698) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:28:03.278+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T09:28:03.281+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:28:03.281+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:28:03.334+0000] {logging_mixin.py:120} INFO - [2024-04-14T09:28:03.332+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T09:28:03.336+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T09:28:03.392+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.146 seconds
[2024-04-14T10:16:19.959+0000] {processor.py:154} INFO - Started process (PID=18765) to work on /opt/airflow/dags/elastic_dag.py
[2024-04-14T10:16:19.962+0000] {processor.py:756} INFO - Processing file /opt/airflow/dags/elastic_dag.py for tasks to queue
[2024-04-14T10:16:19.964+0000] {logging_mixin.py:120} INFO - [2024-04-14T10:16:19.964+0000] {dagbag.py:537} INFO - Filling up the DagBag from /opt/airflow/dags/elastic_dag.py
[2024-04-14T10:16:19.982+0000] {logging_mixin.py:120} INFO - [2024-04-14T10:16:19.980+0000] {dagbag.py:342} ERROR - Failed to import: /opt/airflow/dags/elastic_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 338, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/elastic_dag.py", line 3, in <module>
    from plugins.hooks.elastic.elastic_hook import ElasticHook
ModuleNotFoundError: No module named 'plugins'
[2024-04-14T10:16:19.984+0000] {processor.py:768} WARNING - No viable dags retrieved from /opt/airflow/dags/elastic_dag.py
[2024-04-14T10:16:20.034+0000] {processor.py:176} INFO - Processing /opt/airflow/dags/elastic_dag.py took 0.084 seconds
